processing/src/main/java/org/apache/druid/query/expression/LookupExprMacro.java

```diff
@@ -54,10 +54,11 @@ public String name()
   @Override
   public Expr apply(final List<Expr> args)
   {
-    validationHelperCheckArgumentCount(args, 2);
+    validationHelperCheckMinArgumentCount(args, 2);
```

Comment:
this should use maybe `validationHelperCheckArgumentRange` since it requires 2 or 3 arguments exactly

Reply:
done